### ISSUE TITLE
Blender crashed when loading STL file

### ISSUE THREAD

**Issue by yaxigeigei** [Center-For-Neurotechnology/CorticalNeuropixelProcessingPipeline#1]
Hi, thanks for sharing the pipeline! I love that you took a simple and practical approach to the motion correction. 
So, I had several issues about STL and Blender

1) I was only able to find an add-on called "STL format", as shown below. I am using Blender 2.93 on Windows 10.
![image](https://user-images.githubusercontent.com/3691074/145631095-762d0771-6bb2-4f67-94d6-fa6f02c4ec3d.png)
2) I was able to use File -> Import -> STL to select the STL file (5.2GB in size). Then the RAM usage started to build up, from 22GB all the way to 64GB over a few minutes. After some ups and downs in RAM usage, Blender eventually crashed (no error report shown).

Do you have any idea what's going on?

**Comment by APaulk** [comment]
Hello, 
Thank you. As to your error, I think it has to do with the size of the file. I just tested loading an .stl file (size: 570 MB) using Blender 2.93.4 on a 16 GB RAM laptop and the used memory never got above 12 GB RAM but it did crash with larger files closer to 1 GB. 

My recommendation could be adjusting the sampling or resolution of the matrix exported to the .stl file. I did have to downsample the LFP to keep the size manageable and not crash Blender. It also helped to reduce the number of channels exported (only 50-100 versus all 384) for the alignment. Finally, I did have to do the tracking in batches in two cases. For instance, in one case, I had to subdivide the recording into 5 parts. Each part was exported as a separate .stl file where I tracked the movement manually and then concatenated the movement across the parts in Matlab. Does that make sense?
Thank you and good luck!
ACP
